src/ui/Forms/MultipleReplace.cs

```diff
@@ -207,7 +208,14 @@ private void MultipleReplace_KeyDown(object sender, KeyEventArgs e)
 
         private void RadioButtonCheckedChanged(object sender, EventArgs e)
         {
-            textBoxFind.ContextMenuStrip = sender == radioButtonRegEx ? FindReplaceDialogHelper.GetRegExContextMenu(textBoxFind) : null;
+            // remove regex context menu from find text box
+            textBoxFind.ContextMenuStrip = null;
+
+            // only get the regex context menu if we select the regex option
+            if (sender == radioButtonRegEx && radioButtonRegEx.Checked)
```

Comment:
Checking if ` radioButtonRegEx.Checked`  is important as if the **regex radio button** was previously selected 
and you click to select another radio button, this method will still be invoked twice which will then call `FindReplaceDialogHelper.GetRegExContextMenu(textBoxFind)` and generate non-necessary allocation!